README.md

```diff
@@ -6,37 +6,107 @@ Half-Life SDK for Xash3D & GoldSource with some fixes.
 
 ### CMake as most universal way
 
-```
-mkdir build && cd build
-cmake ../
-```
+    mkdir build && cd build
+    cmake ../
+    make
 
 You may enable or disable some build options by -Dkey=value. All available build options are defined in CMakeLists.txt at root directory.
+See below if you want to build the GoldSource compatible libraries.
 
 See below, if CMake is not suitable for you:
 
 ### Windows
 
+#### Using msvc
+
+We use compilers provided with Microsoft Visual Studio 6. There're `compile.bat` scripts in both `cl_dll` and `dlls` directories.
+Before running any of those files you must define `MSVCDir` variable which is the path to your msvc installation.
+
+    set MSVCDir=C:\Program Files\Microsoft Visual Studio
+    compile.bat
+
+These scripts also can be ran via wine:
+
+    MSVCDir="z:\home\$USER\.wine\drive_c\Program Files\Microsoft Visual Studio" wine cmd /c compile.bat
+
+The libraries built this way are always GoldSource compatible.
+
+There're dsp projects for MVS 6 in `cl_dll` and `dlls` directories, but we don't keep them up-to-date. You're free to adapt them for yourself and try to import into the newer MVS versions.
```

Comment:
Why not to remove them at all? It's an outdated garbage and can be removed.

Reply:
People may still want to use an IDE.
Btw we also could describe how to load CMakeLists.txt in Qt Creator.

Reply:
Qt Creator natively supports CMake. Though it would be nice to describe which IDEs support CMake. I know only 3 IDEs: CLion, KDevelop and QtC, ofc.

Reply:
Some work is required to setup Qt Creator to work with cmake from chroot.